### ISSUE TITLE
BUG?: Files Can't Code Formatting under Chinese folder

### ISSUE THREAD

**Issue by imhuay** [microsoft/vscode-cpptools#870]
VSCode Version: 1.13.1
cpptools Version: 0.12.0
OS Version: Win10_1703
```
workspace
-- test_1.cpp
-- EN_folder
  -- test_2.cpp
-- 中文_文件夹
  -- test_3.cpp
```
`中文_文件夹` means `Chinese_folder`

the `test_1.cpp` and `test_2.cpp` can Code Formatting
but `test_3.cpp` can't Code Formatting

**Comment by sean-mcmanus** [comment]
A fix will be in our pending 0.12.1 update (targeted for some time next week). It was a bug in our Windows CreateChildProcess code (it doesn't repro on Mac/Linux).

**Comment by sean-mcmanus** [comment]
Fix has been released.
